_translator-files/po/de/Client-Troubleshooting.po

```diff
@@ -45,228 +47,230 @@ msgid ""
 
 #. type: Plain text
 #: ../wiki/en/Client-Troubleshooting.md:43
 msgid "Click the \"Mute\" button under your Jamulus name/slider in the main Jamulus mixer window (**NOT** the \"Mute Myself\" button on the left)."
-msgstr ""
+msgstr "Klicke im Hauptfenster des Jamulus-Mixers auf die Schaltfläche \"MUTE\" unter deinem Jamulus-Namen/Slider (**NICHT** die Schaltfläche \"Stummschalten\" auf der linken Seite)."
 
 #. type: Plain text
 #: ../wiki/en/Client-Troubleshooting.md:44
 msgid "Clap your hands again."
-msgstr ""
+msgstr "Klatsche noch einmal in die Hände."
 
 #. type: Plain text
 #: ../wiki/en/Client-Troubleshooting.md:46
 msgid "If you hear two claps after step 2, or any claps after step 4, then you are **not** obeying Rule Number One - you have your local audio enabled and should disable it."
-msgstr ""
+msgstr "Wenn du nach Schritt 2 zwei klatschende Geräusche oder nach Schritt 4 ein Klatschen hörst, dann hälst du dich **nicht** an Regel Nummer eins - Du hast dein lokales Audio aktiviert und solltest es deaktivieren."
 
 #. type: Plain text
 #: ../wiki/en/Client-Troubleshooting.md:48
 msgid "Exactly how you avoid listening to your direct signal will depend on your individual setup - your sound interface, mixing desk, headphone connection point, etc. For example, some audio interfaces have \"monitor\" buttons (turn these off), or similar options. **If you are still having problems**, try asking on the [forum](https://github.com/jamulussoftware/jamulus/discussions)."
-msgstr ""
+msgstr "Wie genau du vermeidest, dass du dein direktes Signal hörst, hängt von deiner individuellen Konfiguration ab - von deinem Audio-Interface, Mischpult, Kopfhöreranschluss usw. Einige Audioschnittstellen haben zum Beispiel \"Monitor\"-Tasten (schalte diese aus) oder ähnliche Optionen. **Wenn du immer noch Probleme hast, kannst du eine Frage im [Forum](https://github.com/jamulussoftware/jamulus/discussions) stellen."
 
 #. type: Plain text
 #: ../wiki/en/Client-Troubleshooting.md:50
 msgid "Be aware that while listening to the Server's signal will ensure you will be in sync with other musicians, you may also experience problems if your overall latency (indicated by the \"Delay\" light in Jamulus) is not green or at least yellow most of the time. Consult the [software manual](/wiki/Software-Manual) to understand how to adjust your setup to help with this."
-msgstr ""
+msgstr "Beachte dass das Abhören des Server Signals zwar sicherstellt, dass du mit den anderen Musikern synchron bist, dass es aber auch zu Problemen kommen kann, wenn die Gesamtlatenz (angezeigt durch die \"Delay\"-Lampe in Jamulus) die meiste Zeit über nicht grün oder zumindest gelb ist. Lies das [Software-Handbuch](/wiki/Software-Manual), um zu erfahren, wie du dein Setup in diesem Fall anpassen kannst."
 
 #. type: Plain text
 #: ../wiki/en/Client-Troubleshooting.md:52
 msgid "### Can't work out your mic settings?"
-msgstr ""
+msgstr "### Du kannst deine Mikrofoneinstellungen nicht herausfinden?"
```

Comment:
--> Probleme mit deinen Mikrofoneinstellungen?
Worth a discussion probably? The current translation sounds very non-native German.

Reply:
Corrected